generator/internal/parser/mixin.go

```diff
@@ -40,15 +40,27 @@ const (
 type mixinMethods map[string]bool
 
 // loadMixins loads file descriptors for configured mixins.
-func loadMixins(serviceConfig *serviceconfig.Service) (mixinMethods, []*descriptorpb.FileDescriptorProto) {
+func loadMixins(serviceConfig *serviceconfig.Service, withLongrunning bool) (mixinMethods, []*descriptorpb.FileDescriptorProto) {
 	var files []*descriptorpb.FileDescriptorProto
 	var enabledMixinMethods mixinMethods = make(map[string]bool)
-	apis := serviceConfig.GetApis()
-	if len(apis) < 2 {
+	var apiNames []string
+	hasLongrunning := false
+	for _, api := range serviceConfig.GetApis() {
+		// Only insert the service if needed. We want to preserve the order
+		// to make the generated code reproducible, so we cannot use a map.
```

Comment:
yeah this is a fun Go quirk. I often find myself getting keys, sorting them, and iterating for the same effect. (what you have is fine)